RInAppMessaging/Classes/Utilities/KeyHasher.swift

```diff
@@ -37,38 +36,21 @@ internal struct KeyHasher {
     private func md5(string: String) -> String {
         let stringData = Data(string.utf8)
 
-        if #available(iOS 13.0, *) {
-            let hash = Insecure.MD5.hash(data: stringData)
-            return hash.hexString
-        } else {
-            var hash = [UInt8](repeating: 0, count: Int(CC_MD5_DIGEST_LENGTH))
-            data.withUnsafeBytes {
-                _ = CC_MD5($0.baseAddress, CC_LONG(data.count), &hash)
-            }
-            return Data(hash).hexString
+        var hash = [UInt8](repeating: 0, count: Int(CC_MD5_DIGEST_LENGTH))
+        stringData.withUnsafeBytes {
+            _ = CC_MD5($0.baseAddress, CC_LONG(stringData.count), &hash)
```

Comment:
Do we need to check in case the stringData.count length is > CC_{ALG}_DIGEST_LENGTH (for MD5 and SHA256 algs) to avoid a possible buffer overrun crash?

Reply:
Hmm I don't think so. You mean when string is longer than hash length?  That should be fine (I added a test for that)

Reply:
Yes, I meant if string is longer than hash buffer. 
OK, I guess CommonCrypto must have a check internally then.